lib/FlagDependencyUsagePlugin.js

```diff
@@ -55,9 +56,15 @@ class FlagDependencyUsagePlugin {
 								} else {
 									let currentExportsInfo = exportsInfo;
 									for (let i = 0; i < usedExport.length; i++) {
-										const exportName = usedExport[i];
+										let exportName = usedExport[i];
+										let canMangle;
+										if (typeof usedExport[i] === "object") {
+											exportName = usedExport[i].name;
+											canMangle = usedExport[i].canMangle;
+										}
 										const exportInfo = currentExportsInfo.getExportInfo(
-											exportName
+											exportName,
+											canMangle
 										);
```

Comment:
Thats a weird way to set canMangleUse in the ModuleGraph with a `get...` method.
`exportInfo.canMangleUse = false;`

``` js
const exportItem = usedExport[i];
let exportInfo;
if (typeof exportItem === "object") {
  exportInfo = currentExportsInfo.getExportInfo(exportItem.name);
  if (exportInfo.canMangle === false) exportInfo.canMangleUse = false;
} else {
  exportInfo = currentExportsInfo.getExportInfo(exportItem);
}
```

Reply:
After invoking `getExportInfo`, we are getting `exportInfo.canMangle` to be `true`. Can I change the `if` condition to the following: `if (exportItem.canMangle === false) exportInfo.canMangleUse = false;`?

Reply:
Oh yes that was a typo

Reply:
okay :)